keyboards/bschwind/key_ripper/info.json

```diff
@@ -0,0 +1,112 @@
+{
+    "manufacturer": "bschwind",
+    "keyboard_name": "key-ripper",
+    "maintainer": "bschwind",
+    "bootloader": "rp2040",
+    "diode_direction": "COL2ROW",
+    "features": {
+        "bootmagic": true,
+        "command": false,
+        "console": false,
+        "extrakey": true,
+        "mousekey": true,
```

Comment:
These were the defaults. Not sure if I should leave them enabled, or disable them.

Reply:
Keeping defaults is fine, its not like you need to reduce firmware size on the rp2040.